src/systems/ec/implementations/vanilla/packages/ec-component-site-header-core/ec-component-site-header-core.scss

```diff
@@ -78,6 +78,12 @@
     }
   }
 
+  .ecl-site-header-core__login-toggle {
+    .ecl-icon--xs {
```

Comment:
Doesn't look quite right to me to be honest. Styling an icon inside current component is kinda scope leak. Although it's been quite long since my last work on these, I think it can be better either to hide elements with utilities or if necessary make a new sub-component element such as `ecl-site-header-core__subelement` to avoid the leak of specificity between components.

Reply:
Speaking here, apparently it's necessary to avoid using utilities classes in components.
Also, there is already a general css rule for the left _logged in_ icon that act on all icons inside that link: https://github.com/ec-europa/europa-component-library/blob/a23635a46490c09fee01cc017d95e10f6b80de0d/src/systems/ec/implementations/vanilla/packages/ec-component-site-header-core/ec-component-site-header-core.scss#L214 this is why I needed to declare properties for xs icone.

Reply:
Although we had to apply css rules on sub elments sometimes, it is indeed better to use a dedicated css class for this.
In this case, we could have something like
`<Icon shape="ui--corner-arrow" size="xs" className="ecl-site-header-standardised__login-icon" />` or something similar. same for the other icon (I don't remember why it has been done like this, but this is a good opportunity to clean it a little).
The trick is that we have to keep the old css rules, to prevent breaking change, but we can mark them as deprecated and remove them in a few version